docs/cpp_standards.md

```diff
@@ -5,10 +5,39 @@ title: Modern C++ standards
 Building Python wheels with modern C++ standards (C++11 and later) requires a few tricks.
 
 
-## manylinux1 and C++14
-The default `manylinux1` image (based on CentOS 5) contains a version of GCC and libstdc++ that only supports C++11 and earlier standards. There are however ways to compile wheels with the C++14 standard (and later): https://github.com/pypa/manylinux/issues/118
+## manylinux and C++
+The `manylinux*` standards imply a limit on which C++ standard you can use. There are workarounds to this, and `cibuildwheel` uses some workarounds by default. You should be aware of them!
```

Comment:
```suggestion
The `manylinux*` standards imply a limit on which C++ standard you can dynamically link to. There is a workaround built into the manylinux1, manylinux2010, and manylinux2014 compilers described below that allows partial linking to just the portions that are allowed; this workaround is not available for the `manylinux_2_*` images yet.
```

For 95%+ of users, this makes no difference - it just works. No one compiles with old GCCs - 4.8 is already _very_ old - C++11 is 10 years old this year. And it's not cibuildwheel's workaround at all, it's Red Hats - one could _maybe_ argue it's manylinux's (which is the way I've stated it above), but it's not cibuildwheel.

Pybind11 would not exist (at least on linux) if we were constrained to the original compilers. And statically linking everything would probably down PyPI.